docs/documents.md

```diff
@@ -0,0 +1,35 @@
+## Rummager Documents API
+
+### `POST /:index/documents`
+
+Insert or overwrite a document.
+
+There must be a link attribute in the JSON body.
+
+#### Example request and response
+
+```json
+{
+    "organisations": [
+      "department-for-transport",
+      "driver-and-vehicle-licensing-agency"
+    ],
+    "public_timestamp": "2014-12-09T16:21:03+00:00",
+    "description": "Renew or tax your vehicle for the first time, apply online, by phone or at the Post Office",
+    "format": "transaction",
+    "link": "/vehicle-tax",
+    "mainstream_browse_pages": [
+      "driving/car-tax-discs"
+    ],
+    "title": "Tax your vehicle",
+    "_type": "edition",
+    "_id": "/vehicle-tax",
+    "specialist_sectors": []
+}
+```
+
+```json
+{
+  "result": "OK"
+}
+```
```

Comment:
Definitely a separate piece of work (and separate ticket, if we decide it's worth doing).